<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MathQuestions</class>
 <widget class="QDialog" name="MathQuestions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>List of questions</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Question1Btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Question 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Question2Btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Question 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Question3Btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Question 3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Question4Btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Question 4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Result">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>460</y>
     <width>201</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Result</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
